core/src/main/java/de/danoeh/antennapod/core/util/ChapterUtils.java

```diff
@@ -135,16 +136,22 @@ private static CountingInputStream openStream(Playable playable, Context context
         }
     }
 
-    public static List<Chapter> loadChaptersFromUrl(String url) {
+    public static List<Chapter> loadChaptersFromUrl(String url, boolean forceRefresh) {
         try {
-            Request request = new Request.Builder().url(url).cacheControl(CacheControl.FORCE_CACHE).build();
+            Request request = null;
+            if (forceRefresh) {
+                request = new Request.Builder().url(url).cacheControl(CacheControl.FORCE_NETWORK).build();
+            } else {
+                request = new Request.Builder().url(url).cacheControl(CacheControl.FORCE_CACHE).build();
+            }
             Response response = AntennapodHttpClient.getHttpClient().newCall(request).execute();
             if (response.isSuccessful() && response.body() != null) {
                 List<Chapter> chapters = PodcastIndexChapterParser.parse(response.body().string());
                 if (chapters != null && !chapters.isEmpty()) {
                     return chapters;
                 }
             }
+
```

Comment:
Please try to keep other code like it is, on order to avoid merge conflicts and a hard-to-read git history. So, please remove this line again.